content/concepts/the-rippled-server/ledger-history/ledger-history.md

```diff
@@ -18,17 +18,21 @@ The [server_info method][] reports how many ledger versions your server has avai
 
 ## Fetching History
 
-When it starts, a `rippled` server's first priority is to get a complete copy of the latest validated ledger. From there, it keeps up with advances in the ledger progress. If configured to do so, the server also backfills ledger history up to a configured amount, which must be equal to or less than the cutoff beyond which online deletion is configured to delete.
+When it starts, a `rippled` server's first priority is to get a complete copy of the latest validated ledger. From there, it keeps up with advances in the ledger progress. The server can also backfill history from before it became synced, as well as filling in any gaps in the history it has collected after syncing. (Gaps in ledger history can occur if a server temporarily becomes too busy to keep up with the network, loses its network connection, or suffers other temporary issues.) To backfill history, the server requests data from its peer servers.
```

Comment:
"to backfill history, the server requests data from its peer server" is this process automatic or does it depend on intervention by a user? only happens when the server becomes synced? happens every 5 min? etc. might be good to include some context here about whether this process happens automatically and its cadence or in the "Backfilling" section